internal/uvm/vpmem.go

```diff
@@ -142,13 +144,20 @@ func (uvm *UtilityVM) RemoveVPMEM(ctx context.Context, hostPath string) (err err
 			return fmt.Errorf("failed to remove VPMEM %s from utility VM %s: %s", hostPath, uvm.id, err)
 		}
 		uvm.vpmemDevices[deviceNumber] = vpmemInfo{}
+		uvm.vpmemNumDevices--
 		return nil
 	}
 	uvm.vpmemDevices[deviceNumber].refCount--
+	uvm.vpmemNumDevices--
```

Comment:
I don't think we would want to decrement `vpmemNumDevices` in this case. We reduced the `refCount`, but still have the same number of devices. Am I missing something?

Reply:
I think you're correct. Will update.